src/util/sourceMappingDecoder.js

```diff
@@ -0,0 +1,132 @@
+'use strict'
+var util = require('../helpers/util')
+
+/*
+  Decompress the source mapping given by solc-bin.js
+*/
+function SourceMappingDecoder () {
+  // s:l:f:j
+}
+
+/*
+    * Decode the source mapping for the given @arg index
+    *
+    * @param {Integer} index      - source mapping index to decode
+    * @param {String} mapping     - compressed source mapping given by solc-bin
+    * @param {Function} callback  - returns the decompressed source mapping for the given index {start, length, file, jump}
+*/
+SourceMappingDecoder.prototype.decompress = function (index, mapping, cb) {
```

Comment:
This function basically performs an index access of the however encoded mapping at a specific point. I think "decompression" rather suggests that it returns the full list. What about `atIndex(mapping, index)` - and also make it synchronous?